### PR TITLE
Asserts and change of bool initFromStream functions to voids.

### PR DESCRIPTION
**Pull request by Sydriax**
Hopefully cleaner and with fewer dependencies.
@truell20 You said in the earlier pull request that my indentation is messed up; I believe you, but I've looked through it on my version and it seems fine. Can you point to where it's a problem?

### CODE REVIEW

File: src/Adadelta.cpp [hunk 1]
```diff
@@ -24,24 +24,15 @@ double Adadelta::getChangeInWeight(double weight, int layerIndex, int neuronInde
...
 
 void Adadelta::store(std::ofstream *output) {
-  if(output->is_open()) {
-		SGDTrainer::store(output);
-		*output << rho << " " << epsilon << "\n";
-	} else {
-		std::cout << "Could not store backprop\n";
-		throw 1;
-	}
+  assert(output->is_open());
```
Comment (truell20) [inline src/Adadelta.cpp]: @Sydriax here

**Comment by truell20** [comment]
@Sydriax Did you see my note?

**Comment by truell20** [comment]
@Sydriax Also, you dont have to resubmit the pull request. You can just push commits and the request will update

File: src/Adadelta.cpp (hunk 1 above)
Comment (Sydriax) [inline src/Adadelta.cpp]: ty

File: src/Layer.cpp [hunk 2]
```diff
@@ -28,19 +30,14 @@ Layer::Layer(std::ifstream *input) {
 void Layer::setActivationFunctionWithName(std::string name) {
 	std::map<std::string, ActivationFunction> nameMap = getActivationFunctionNameMap();
 	std::map<std::string, ActivationFunction>::iterator nameIterator = nameMap.find(name);
-	if(nameIterator != nameMap.end()) {
-		activationFunction = getActivationFunctionNameMap()[name];
-	} else {
-		std::cout << "The activation function \"" << name << "\" does not exist! Throwing error!\n";
-		throw 1;
-	}
+	assert(nameIterator == nameMap.end());
```
Comment (truell20) [inline src/Layer.cpp]: Should be a `!=`

**Pull request merged**